### ISSUE TITLE
Polycyclic Thermo Corrections for Aromatics Requires Benzene and Kekulized Forms

### ISSUE THREAD

**Issue by zjburas** [ReactionMechanismGenerator/RMG-Py#884]
The following strained aromatic-containing bicyclic species was appearing in much too high concentrations in RMG due to a poor thermo estimate:

![image](https://cloud.githubusercontent.com/assets/12465817/22676711/190c12d8-ecbb-11e6-9466-418dbaba8c18.png)

I calculate the thermo of this molecule at the M06 level of theory, and made a new polycyclic thermo correction for it using benzene bonds to define the group. However, when I tested RMG's new thermo estimation for this molecule, it did not use the new polycyclic group that I had created, but instead estimated the thermo based on its kekulized form:

![image](https://cloud.githubusercontent.com/assets/12465817/22676915/61c77c3c-ecbc-11e6-9ae0-f42ec10a1ebb.png)

which of course used totally different groups  and a generic polycyclic correction that once again underestimated how strained this molecule is. Only after making a second new polycyclic correction, this time for the kekulized structure, were the intended thermo corrections accessed.

Basically, whenever you make a polycyclic thermo correction for an aromatic containing molecule, you have to create at least two: one for the benzene bond form of the molecule, and another for the kekulized form. 

This is a known issue, and after talking with @KEHANG about it, he mentioned that @connie had a hack  around this issue on some branch that has not been merged with master.

**Comment by zjburas** [comment]
I think this is still an issue.

**Comment by mliu49** [comment]
This has finally been fixed in #1731. The approach taken was to prioritize resonance structures with more aromatic rings.
